### ISSUE TITLE
Select multiple tasks at once with keyboard and mouse in Grid View

### ISSUE THREAD

**Issue by qcha41** [apache/airflow#33448]
### Description

I think it would be nice to have the ability to select multiple tasks (to clean or to set success or failed) at once in the Grid View by :
- Drawing a square with the mouse
- Click on a task, hold CTRL keyboard button, click a second task somewhere else -> two tasks selected
- Click on a task, hold SHIFT keyboard button, click a second task somewhere else -> a square a tasks selected, first task being at the upper left, the second at the lower right 

### Use case/motivation

In some cases it would be really simpler and more efficient to select tasks this way in the grid view, instead of crafting a request in the Browse panel, iterating over the pages if several, etc.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by bbovenzi** [comment]
Good idea! I actually thought we already had an issue for this, but I guess not.

**Comment by Adaverse** [comment]
Could be a part of this #31556

**Comment by RodrigoGanancia** [comment]
Could @jonhspyro and I take care of this issue?

**Comment by gaelxcowi** [comment]
is there a status on this?

**Comment by pierrejeambrun** [comment]
@gaelxcowi There was an attempt here https://github.com/apache/airflow/pull/40147 but got closed. As you can see there are a few implications about updating the details pane as well as adding bulk actions on tasks. This is not super trivial.

I don't think anybody is working on this issue at the moment, feel free to take it.

Also from a release point of view such new feature can only make it into airflow 3. (next minor release is a bridge release without additional features)

**Comment by gaelxcowi** [comment]
@pierrejeambrun understood, thank you for the status.

**Comment by potiuk** [comment]
@RodrigoGanancia We are unassigning you from this issue as part of our updated [assignment policy](https://github.com/apache/airflow/blob/main/contributing-docs/04_how_to_contribute.rst#contribute-code-changes).

This is not meant to discourage your contribution — quite the opposite! You are still very welcome to work on this issue and submit a PR for it. Simply comment that you are working on it and open a PR when ready.

We found that formal assignments were not working well, as they often prevented others from contributing when the assignee was not actively working on the issue.
